package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "eslint-plugin-mocha",
-    "version": "10.0.3",
+    "version": "10.0.4",
```

Comment:
Please don’t increase the package version in this PR. Releases will be done from the `master` branch once all changes have been merged.

Reply:
Done

Reply:
Thank you.